iothub/device/tests/FxTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.Client.Tests
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class FxTests
+    {
+        [TestMethod]
+        public void IotHubDeviceClient_FatalException()
+        {
+            OutOfMemoryException exception = new ();
```

Comment:
It seems the .NET style is to have no space before the parens.

```suggestion
            OutOfMemoryException exception = new();
```